cyclonedds/idl/_xt_builder.py

```diff
@@ -27,6 +25,9 @@
 from ._type_normalize import get_extended_type_hints, get_idl_annotations, get_idl_field_annotations, WrapOpt
 
 
+uint32_max = 2 ** 32 - 1
```

Comment:
I don't Python all that well, so forgive me for asking: is this going to be fine on a 32-bits platform?

Reply:
Python integer types are effectively BigInt implementations with unlimited precision, so you can keep raising powers till you run out of memory 😄.